bundles/org.openhab.ui.dashboard/README.md

```diff
@@ -0,0 +1,34 @@
+# Dashboard tiles
+
+OpenHAB dashboard is landing page for the user where all openHAB UI's can be found. Dashboard support also links to external services. Links can be added to dashboard by ```conf/services/dashboard.cfg``` configuration file.
+
+## Link Configuration
+
+| Parameter name  | Type    | Description                                                                             |
+|-----------------|---------|-----------------------------------------------------------------------------------------|
+| link-nameX      | String  | Name which is shown in the openHAB dashboard.                                           |
+| link-urlX       | String  | URL to external service.                                                                |
+| link-overlayX   | String  | Image overlay icon. Supported values are empty (no icon), "html5", "android" or "apple" |
+| link-imageurlX  | String  | URL to image.                                                                           |
+
+Where X is link unique number (see examples). All configuration parameters need to start with ```org.openhab.ui.dashboard:``` prefix.
+
+## Image URL
+
+Image URL support several URL formats. URL support direct http links to local or Internet servers, but also the "data" URL scheme (RFC2397) is supported. See e.g. [https://www.base64-image.de](https://www.base64-image.de) to convert images to base64 coded data.
+
+## Example configuration file
+```
+org.openhab.ui.dashboard:link-name1=openHAB Log Viewer
+org.openhab.ui.dashboard:link-url1=http://localhost:9001
```

Comment:
If the answer to my question above is "the browser", your examples are pretty misleading - "localhost" won't really get you far...